coralproject/talk
# Added asset merging + url updates

## What does this PR do?

Adds new backend services for merging assets and updating asset url's.

- Added new `cli assets merge <srcID> <dstID>` command
- Added new `cli assets update-url <id> <url>` command

## How do I test this PR?

1. Create a few asset streams
2. Try to merge those asset streams (at least 2 of them)
3. Change a url of another asset

Notice no errors! =D
